glide.yaml

```diff
@@ -17,9 +15,16 @@ import:
 - package: github.com/giantswarm/microstorage
   version: master
 - package: github.com/giantswarm/operatorkit
-  version: kubernetes-1.7
+  version: master
+
+- package: k8s.io/api
+  version: kubernetes-1.8.5
+- package: k8s.io/apiextensions-apiserver
+  version: kubernetes-1.8.5
+- package: k8s.io/apimachinery
+  version: kubernetes-1.8.5
+- package: k8s.io/client-go
+  version: kubernetes-1.8.5
 
-- package: github.com/coreos/etcd
-  version: 61fc123e7a8b14a0a258aa3f5c4159861b1ec2e7
-- package: golang.org/x/net
-  version: f2499483f923065a842d38eb4c7f1927e6fc6e6d
+- package: github.com/coreos/go-semver
+  version: 8ab6407b697782a06568d4b7f1db25550ec2e4c6
```

Comment:
All this is some weirdness I hope to get rid of when moving to use `dep` in the next step. Glide was also retarded here. It gave me some code from some commit from 2015. This hash here is the commit of the latest tag `v0.2.0`. :table-flip:.